library/kotlin/src/io/envoyproxy/envoymobile/BUILD

```diff
@@ -30,3 +30,13 @@ kt_android_library(
     visibility = ["//visibility:public"],
     deps = ["//library/java/io/envoyproxy/envoymobile/engine:envoy_engine_lib"],
 )
+
+kt_jvm_library(
+    name = "envoy_interfaces_lib",
```

Comment:
Shouldn't this be part of `envoy_lib`?

Reply:
It'll pull in all the dependencies with the JNI, Android, etc. I wanted to keep this more vanilla kotlin for now because I hope to isolate most of the classes away from Android and Envoy. The rationale is because I want to keep the tests lighter weight and we can more easily make decisions independent from the other layers.

We can also revisit the organization and responsibility boundaries down the line